src/test/util_tests.cpp

```diff
@@ -21,6 +21,26 @@
 
 #include <boost/test/unit_test.hpp>
 
+namespace util_tests {
+    // Create and return temporary subdirectory for test outputs
+    fs::path unit_test_directory(const std::string fileName){
+
+        std::string sub_dir = "bitcoin_unit_tests";
+        fs::path dir(fs::temp_directory_path() / sub_dir);
+        fs::path test_dir(dir / fileName / std::string(boost::unit_test::framework::current_test_case().p_name));
+
+        if(!fs::exists(dir))
```

Comment:
The dir will exist when the previous run was not shut down cleanly or when running two `test_bitcoin` at the same time. I don't think we should silently continue here.